audit-base/src/main/java/com/wultra/core/audit/base/database/DatabaseAuditWriter.java

```diff
@@ -281,12 +286,14 @@ public void scheduledFlush() {
     }
 
     /**
-     * Scheduled cleanup of audit data in database.
+     * Scheduled cleanup of audit data in the database.
      */
-    @Scheduled(fixedDelayString = "${audit.cleanup.delay.fixed:3600000}", initialDelayString = "${powerauth.audit.cleanup.delay.initial:1000}")
+    @Scheduled(cron = "${audit.cleanup.cron:0 59 23 * * SAT}")
+    @SchedulerLock(name = "audit.cleanup")
```

Comment:
The lock least ad lock most setting is needed. 
LockLeast - to prevent ultra quick duplicity running on multiple nodes
LockMost is required to prevent the stalling of the job.

Reply:
A global `defaultLockAtMostFor` is defined as a required parameter of `@EnableSchedulerLock`. Is it needed to override that?

Reply:
The same situation is for `lockAtLeastFor`, we may use an optional `defaultLockAtLeastFor`.

Reply:
🤔 But for Enrollnem, current config is

```java
@EnableSchedulerLock(defaultLockAtLeastFor = "100ms", defaultLockAtMostFor = "1m")
```
Ok, let's use explicit values.